apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.html

```diff
@@ -33,7 +35,7 @@
             />
         </mat-chip-grid>
         <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selectOption($event); selectField.value = ''">
-            <mat-option [ngClass]="{'po-options-line': isPurchaseOrder }" *ngFor="let option of fieldFilteredOptions" [value]="option">
+            <mat-option (click)="(name === emailType.To || name === emailType.Cc || name === emailType.Bcc) && $event.stopPropagation()" [ngClass]="{'po-options-line': isPurchaseOrder }" *ngFor="let option of fieldFilteredOptions" [value]="option">
```

Comment:
what's the use of this click event

Reply:
When the user clicks on the dropdown of the email field, the clickOutside function is automatically triggered. I have stopped this event specifically for the email field